iBudget/static/src/components/examples/Chart_spending.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+    <div class="container">
+        <div class="Chart">
+            <h1 style="text-align:center;">This month spending</h1>
+            <pie-example v-bind:pieData="value" v-bind:pieLabel="name"/>
+        </div>
+    </div>
+</template>
+
+<script>
+    import PieExample from './PieExample';
+
+    export default {
+        components: {
+            PieExample,
+        },
+        props: ['value', 'name'],
+        data() {
+            return {
+                height: 80,
+            }
+        },
+        computed: {
+            myStyles() {
+                return {
+                    height: `${this.height}px`,
+                    position: 'relative'
+                }
+            }
+        }
+    }
+</script>
+
+<style>
+    .container {
+        max-width: 600px;
+        margin: auto;
+    }
+
+    h1 {
+        font-family: 'Helvetica', Arial;
```

Comment:
It it better to add type of font at the end, like: `sans-serif` or `serif`,

Reply:
Ok. I'll improve it